docker/.dev.env

```diff
@@ -0,0 +1,2 @@
+MINIO_ACCESS_KEY=minio
+MINIO_SECRET_KEY=minio123
```

Comment:
I decided to go with an env file that has credentials on it **just** because this is for DEV and scope is limited to a developer's dev environment. Having the credentials directly on the `docker-compose.yml` meant having to input them twice: once for `minio` container and again for `setup-initial-bucket` container. This way, we only have them defined once and then referenced by both containers.